specification.md

```diff
@@ -897,13 +897,40 @@ debugging purposes, but which should not, in general, be presented to an
 end-user.
 
 
-<a name="invalidjobexception-getjob"></a>
+
+### SubmitException
+
+This exception is thrown when the
+[`JobExecutor.submit()`](#jobexecutor-submit) call fails for a reason
+that is independent of the job being submitted.
```

Comment:
Sorry. Really minor nit but this was confusing to me on the second read through.  As written, it kind of sounds like "call fails for a reason that is independent of the job submission".  My suggestion:

```suggestion
that is independent of the job that is being submitted.
```